Detections/AzureDiagnostics/KeyvaultMassSecretRetrieval.yaml

```diff
@@ -47,4 +47,5 @@ query: |
   | where OperationName in (OperationList)   
   ) on identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g 
   | project TimeGenerated, ResourceType, ResultType, Resource, OperationName, id_s, CallerIPAddress, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, requestUri_s, clientInfo_s
-  | extend timestamp = TimeGenerated, IPCustomEntity = CallerIPAddress, AccountCustomEntity = identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g
\ No newline at end of file
+  | summarize count(), TimeGeneratedMax=arg_max(TimeGenerated,*), TimeTriggered=makelist(TimeGenerated),OperationNameList=make_set(OperationName), RequestURLList=make_set(requestUri_s), CallerIPList = make_set(CallerIPAddress),  CallerIPMax= arg_max(CallerIPAddress,*) by ResourceType, ResultType, Resource, id_s, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, clientInfo_s
```

Comment:
Same as above, lets label what the count() is related to.

Reply:
Also, you are doing a count above the join, should you be using sum(count_) to make sure the correct count comes thru?